README.md

```diff
@@ -2629,6 +2693,218 @@ Defines haptic data for each user control object for video streaming application
+|`queuePlaybackDuration`|Integer|False|Music: The total duration of the playback queue in seconds                Podcast: The total duration of the playback queue in seconds                Audiobook: The total duration of the playback queue (e.g. the book) in seconds            |
+|`queueCurrentTrackNumber`|Integer|False|Music: The current number (1 based) of the track in the playback queue                Podcast: The current number (1 based) of the episode in the playback queue                Audiobook: The current number (1 based) of the episode in the playback queue (e.g. the chapter number in the book)            |
+|`queueTotalTrackCount`|Integer|False|Music: The total number of tracks in the playback queue                Podcast: The total number of episodes in the playback queue                Audiobook: The total number of sections in the playback queue (e.g. the number of chapters in the book)            |
+
+
+### WeatherServiceManifest
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`currentForecastSupported`|Boolean|False||
+|`maxMultidayForecastAmount`|Integer|False||
+|`maxHourlyForecastAmount`|Integer|False||
+|`maxMinutelyForecastAmount`|Integer|False||
+|`weatherForLocationSupported`|Boolean|False||
+
+
+### WeatherAlert
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`title`|String|False||
+|`summary`|String|False||
+|`expires`|DateTime|False||
+|`regions`|String[]|False||
+|`severity`|String|False||
+|`timeIssued`|DateTime|False||
+
+
+### WeatherData
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`currentTemperature`|Temperature|False||
+|`temperatureHigh`|Temperature|False||
+|`temperatureLow`|Temperature|False||
+|`apparentTemperature`|Temperature|False||
+|`apparentTemperatureHigh`|Temperature|False||
+|`apparentTemperatureLow`|Temperature|False||
+|`weatherSummary`|String|False||
+|`time`|DateTime|False||
+|`humidity`|Float|False|0 to 1, percentage humidity |
+|`cloudCover`|Float|False|0 to 1, percentage cloud cover |
+|`moonPhase`|Float|False|0 to 1, percentage of the moon seen, e.g. 0 = no moon, 0.25 = quarter moon |
+|`windBearing`|Integer|False|In degrees, true north at 0 degrees |
+|`windGust`|Float|False|km/hr |
+|`windSpeed`|Float|False|km/hr |
+|`nearestStormBearing`|Integer|False|In degrees, true north at 0 degrees |
+|`nearestStormDistance`|Integer|False|In km |
+|`precipAccumulation`|Float|False|cm |
+|`precipIntensity`|Float|False|cm of water per hour |
+|`precipProbability`|Float|False|0 to 1, percentage chance |
+|`precipType`|String|False|e.g. "rain", "snow", "sleet", "hail" |
+|`visibility`|Float|False|In km |
+|`weatherIconImageName`|String|False||
```

Comment:
Update so the name is `weatherIcon` and the data type is `Image`

Reply:
Fixed 👍